components/apim-analytics/org.wso2.analytics.solutions.apim.analytics/widgets/APIMApiCreated/.eslintrc.js

```diff
@@ -0,0 +1,70 @@
+module.exports = {
```

Comment:
Sice this is a JS files, should have the license header.

Reply:
Fixed.